src/compiler/diagnosticMessages.json

```diff
@@ -1687,6 +1691,10 @@
         "category": "Error",
         "code": 2528
     },
+    "Duplicate identifier '{0}'. Compiler reserves name '{1}' in top level scope of a module containing async functions.": {

```

Comment:
The compiler reserves '{1}' in the top level scope of of a module when using async functions.